src/components/utils/caseUtils.ts

```diff
@@ -1,4 +1,14 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
 import { downloadCase, getCaseOriginalName } from '../../utils/rest-api';
+import { ElementType } from '../../utils/elementType';
+import { useIntl } from 'react-intl';
+import { useSnackMessage } from '@gridsuite/commons-ui';
 
 export const downloadCases = async (uuids: string[]) => {
```

Comment:
If it's used only locally now (because you export useDownloadUtils now) maybe remove this export here

Reply:
Yes, done